dev-python/spyder/spyder-5.0.1.ebuild

```diff
@@ -0,0 +1,202 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} )
+# The warning that this is wrong is a false positive
+# Spyder has setuptools in install_requires
+DISTUTILS_USE_SETUPTOOLS=rdepend
```

Comment:
Is there still a warning, though? If yes, please report a bug. The check is supposed to handle `install_requires`.

Reply:
No this seems to have been fixed now, nice :)